smarts/core/agent_interface.py

```diff
@@ -188,7 +188,7 @@ class DoneCriteria:
     """End the episode when the agent drives in the wrong direction, even though it
     may be driving on the mission route.
     """
-    not_moving: bool = False
+    not_moving: Tuple = (False, 60)
```

Comment:
I think we have run into a semantic issue here. This is no longer simply `DoneCriteria`. This is full event configuration, these events still fire even if they do not cause done so changing the time it takes for the `not_moving` event to fire is independent from if it is flagged as a done criteria.

One of the following must be done here:
- change this class into an `EventConfiguration`
- have `EventConfiguration` separate from the `DoneCriteria`.